docs/motivation.md

```diff
@@ -277,6 +277,26 @@ assumes the existence of a coalescing queue on input. The client SHOULD NOT
 assume that transactions for different paths are processed in the order in
 which they are received.
 
+### Persistence of Configuration
+
+Like other control-plane protocols, the "dynamic" configuration of a device
+is not expected to be persisted across device reboots. Data that is written
+via gNMI to a `config` path is expected to be persisted after each transaction.
```

Comment:
and across reloads

Reply:
Persisted after each transaction means that it's persisted across reloads. Not sure this clarification is needed. I've made a note that this storage it is persisted to is non-volatile which seems like the only clarification needed.